src/ports/postgres/modules/graph/hits.sql_in

```diff
@@ -103,18 +102,18 @@ It is named by adding the suffix '_summary' to the 'out_table'
 parameter.
 
 <dt>max_iter (optional) </dt>
-<dd>INTEGER, default: 100. The maximum number of iterations allowed. An 
+<dd>INTEGER, default: 100. The maximum number of iterations allowed. One 
     iteration consists of both authority and hub phases.</dd>
 
 <dt>threshold (optional) </dt>
 <dd>FLOAT8, default: (1/number of vertices * 1000).
+    Threshold must be set to a value between 0 and 1. 
```

Comment:
Is this inclusive of end points?

Reply:
yes, range is [0,1]
0 means run full num_iter specified

Will clarify in docs